pom.xml

```diff
@@ -28,7 +28,7 @@
     <concurrency>2</concurrency>
     <!-- TODO: Ongoing work on https://github.com/MarkEWaite/git-plugin/tree/master-findbugs-fixes. -->
     <findbugs.failOnError>false</findbugs.failOnError>
-    <workflow.version>1.14.2-SNAPSHOT</workflow.version> <!-- TODO https://github.com/jenkinsci/pipeline-plugin/pull/389 -->
+    <workflow.version>1.14.2</workflow.version>
```

Comment:
How can I find the jenkins.version dependency of workflow.version 1.14.2?  I can't seem to find the 1.14.2 release of workflow-scm-step-plugin, and thus can't read its pom.

If this change will make the dependency 1.625 or newer, then I'll need to wait to merge it to the 3.0.0-beta branch.  If the change remains at 1.609 or earlier, then I can merge it to the master branch.